grow/deployments/destinations/webreview_destination.py

```diff
@@ -100,7 +100,7 @@ def deploy(self, *args, **kwargs):
                 url = finalize_response['fileset']['url']
                 # Append the homepage path to the staging link.
                 result = urlparse.urlparse(url)
-                if not result.path and self.pod.get_home_doc():
+                if not result.path and self.pod and self.pod.get_home_doc():
```

Comment:
Yes.  It can happen if you don't have a webreview deployment in your podspec.

  File "snip/src/grow/bin/../grow/deployments/destinations/webreview_destination.py", line 103, in deploy
    if not result.path and self.pod.get_home_doc():
AttributeError: 'NoneType' object has no attribute 'get_home_doc'